core/src/main/java/com/epam/esm/util/converter/EntityConverter.java

```diff
@@ -0,0 +1,75 @@
+package com.epam.esm.util.converter;
+
+import com.epam.esm.model.dto.GiftCertificateDto;
+import com.epam.esm.model.dto.TagDto;
+import com.epam.esm.model.entity.GiftCertificateEntity;
+import com.epam.esm.model.entity.TagEntity;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+
+import java.sql.Timestamp;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class EntityConverter {
+
+    public static GiftCertificateDto convertGiftEntityDto(GiftCertificateEntity source) {
+        Timestamp createDate = source.getCreateDate();
+        Timestamp lastUpdateDate = source.getLastUpdateDate();
```

Comment:
could you please create this variables below after tag's converting
it is a bit confusing as I would expect that these fields are used for tag's converting but they aren't.